tests/executors/test_celery_executor.py

```diff
@@ -529,7 +529,7 @@ def test_send_tasks_to_celery_hang(register_signals):
     executor = celery_executor.CeleryExecutor()
 
     task = MockTask()
-    task_tuples_to_send = [(None, None, None, None, task) for _ in range(26)]
+    task_tuples_to_send = [(None, None, None, task) for _ in range(26)]
```

Comment:
`_send_tasks_to_celery` was modified to accept 4-tuples (not 5) a while ago, but this test was never updated. Not related to AIP-39.